### PR TITLE
New: Kit defaults - Usage data [ED-8178]

### PR DESCRIPTION
**Pull request by StyleShit**

### CODE REVIEW

**Review by Nevoss — DISMISSED** [review]

File: modules/kit-elements-defaults/module.php [hunk 1]
```diff
@@ -45,5 +48,7 @@ public function __construct() {
 		} );
 
 		Plugin::$instance->data_manager_v2->register_controller( new Controller() );
+
+		( new Usage() )->register();
```
Comment (matipojo) [inline modules/kit-elements-defaults/module.php:52]: if is_admin

File: modules/kit-elements-defaults/module.php [hunk 2]
```diff
@@ -11,6 +11,9 @@
 }
 
 class Module extends BaseModule {
+
+	const META_KEY = '_elementor_elements_defaults_values';
```
Comment (matipojo) [inline modules/kit-elements-defaults/module.php:15]:
```suggestion
	const META_KEY = '_elementor_elements_default_values';
```

**Review by matipojo — CHANGES_REQUESTED** [review]

File: modules/kit-elements-defaults/module.php (hunk 2 above)
Comment (StyleShit) [reply modules/kit-elements-defaults/module.php:15]: Good catch.
It was copy-pasted from the controller, so we'll need to fix that in the other PRs @Nevoss

File: modules/kit-elements-defaults/module.php (hunk 1 above)
Comment (StyleShit) [reply modules/kit-elements-defaults/module.php:52]: I didn't find any other place in the code where we check that before sending the tracking data...

**Pull request merged**